prow/github/types.go

```diff
@@ -122,11 +122,21 @@ type Label struct {
 	Color string `json:"color"`
 }
 
+// PullRequestFileStatus enumerates the statuses for this webhook payload type.
+type PullRequestFileStatus string
+
+const (
+	PullRequestFileModified PullRequestFileStatus = "modified"
```

Comment:
I didn't find statuses on github API pages https://developer.github.com/v3/.

Just found something here https://github.com/github/VisualStudio/blob/5baf8379abdc0356dffa356f2d66855aaedf063f/src/GitHub.Exports/Models/IPullRequestFileModel.cs

```
    public enum PullRequestFileStatus
    {
        Modified,
        Added,
        Removed,
        Renamed,
    }
```

I think this is what we want.